README.md

```diff
@@ -1,5 +1,12 @@
-Project Name: ANIMEVERSE
-
-DESCRIPTION:-
-
-In this project, we will have a diverse range of anime shows, with each show belonging to a unique genre. Our objective is to create an exhaustive list from the most popular and critically acclaimed anime series to the hidden gems that need more recognition.
+
+Project Name: ANIMEVERSE

```

Comment:
The content in README.md is duplicated. It's better to keep the README concise and avoid duplication. [medium]